test/BUILD.gn

```diff
@@ -326,6 +326,17 @@ test("brave_unit_tests") {
     ]
   }
 
+  if (brave_wallet_enabled) {
+    sources +=
+        [ "//brave/browser/brave_wallet/brave_wallet_tab_helper_unittest.cc" ]
```

Comment:
This should be added to the unit_tests target in `brave/browser/brave_wallet/BUILD.gn` instead.

Reply:
Fixed in 4b1db3a7dc283d1ca75a6640a09128a28b1bc367